commands/manifest/template.go

```diff
@@ -83,19 +83,10 @@ func templateCmd(c *cli.Context) error {
 		return fmt.Errorf("cannot resolve manifest vars %s", err.Error())
 	}
 
-	if strings.HasPrefix(m.Chart.Name, "git@") ||
-		strings.Contains(m.Chart.Name, ".git") { // for https:// git urls
-		tmpChartDir, err := helm.CloneChartFromRepo(m, "")
-		if err != nil {
-			return fmt.Errorf("cannot fetch chart from git %s", err.Error())
-		}
-		m.Chart.Name = tmpChartDir
-		defer os.RemoveAll(tmpChartDir)
-	}
-
-	templatesManifests, err := helm.HelmTemplate(m)
+	// Get templates manifests
+	templatesManifests, err := getTemplatesManifests(m)
```

Comment:
I believe the `templatesManifests` was originally a typo: `templatedManifests` is what expresses the variable contents